ros_buildfarm/templates/release/release_script.sh.em

```diff
@@ -47,8 +47,13 @@ cd $WORKSPACE
 
 echo ""
 echo "Get artifacts from source job"
+@[if package_format == 'deb']@
 mkdir -p $WORKSPACE/binarydeb
 (set -x; cp $BASEPATH/source/sourcedeb/*.debian.tar.[gx]z $BASEPATH/source/sourcedeb/*.dsc $BASEPATH/source/sourcedeb/*.orig.tar.gz $WORKSPACE/binarydeb/)
+@[else]@
+mkdir -p $WORKSPACE/binarypkg/source
+(set -x; cp $BASEPATH/source/sourcepkg/* $WORKSPACE/binarypkg/source/)
```

Comment:
The original change only copied the source RPM forward. The only other files that `mock` creates in the output directory are log files. From an RPM perspective, they're safe to propagate. Obviously I can't speak for future non-debian/non-RPM platforms, though.

Reply:
I think that rather than a universal glob copying everything I'd probably copy just the files needed and then add additional globs for new platforms. As an example if deb and rpms both used binarypkg:

```
(set -x; cp $BASEPATH/source/sourcepkg/*.srpm $BASEPATH/source/sourcepkg/*.debian.tar.[gx]z $BASEPATH/source/sourcepkg/*.dsc $BASEPATH/source/sourcepkg*.orig.tar.gz $WORKSPACE/binarypkg/source/)
```

It's a little less future proof but I have a slight preference the explicit approach. This does however mean that if ever there is a file required by platform FOO and generated but not required by platform BAR then this will still bring extraneous files.

I'll let you as the author make the final decision.

Reply:
I modified it to target the source RPM specifically in 0aa0f79. If/when a new packaging format shows up, there will be an assert to let us know that we need to add to the conditional.